index.html

```diff
@@ -11,9 +11,66 @@
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
+
+  <nav>
+    <a href="#">Home</a>
+    <a href="#">About</a>
+    <a href="#">Portfolio</a>
+    <a href="#">Contact</a>
+    </nav>
 </head>
+<div class="container">
+<div class="image1">
+  <img src="Images/beach.jpg" alt="beach pic"/>
```

Comment:
this probably worked locally on your machine but on github when I previewed it was confused by the capital i in Images/beach. changing it to a lowercase fixed it right up. something to watch out for